<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1884</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_000000">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_bbbbbb">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_ffffff">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_49423d">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_ff0000">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_ffa500">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_0000ff">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_ffff00">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_964b00">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_00ffff">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_8b00ff">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_ffc0cb">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">color_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_person_color">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="thick_slider">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="txt_thick_label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Thick:</string>
   </property>
  </widget>
  <widget class="QLabel" name="value_thick_label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>5px</string>
   </property>
  </widget>
  <widget class="QLabel" name="personal_color_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your color</string>
   </property>
  </widget>
  <widget class="QLabel" name="color_label">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Colors</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stylus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="tools_label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>109</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_alpha_label">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Opacity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="value_alpha_label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QSlider" name="alpha_slider">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>80</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="marker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="selected_color">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>60</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="is_used_label_tool">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>marker</string>
   </property>
  </widget>
  <widget class="QLabel" name="is_used_label">
   <property name="geometry">
    <rect>
     <x>1224</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Using:</string>
   </property>
  </widget>
  <widget class="QLabel" name="is_used_label_px">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5 px</string>
   </property>
  </widget>
  <widget class="QLabel" name="is_used_label_alpha">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brush">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>40</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rubber">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="rectangle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="btn_fill">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_color_fill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="left_btn">
   <property name="geometry">
    <rect>
     <x>1640</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right_btn">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>--&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="left_lbl">
   <property name="geometry">
    <rect>
     <x>1640</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="right_lbl">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вперёд</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save_as_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="base_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Base</string>
   </property>
  </widget>
  <widget class="QPushButton" name="circle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">tool_group</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tool_group"/>
  <buttongroup name="color_group"/>
 </buttongroups>
</ui>
